Sources/Views/MessagesCollectionView.swift

```diff
@@ -100,8 +100,24 @@ open class MessagesCollectionView: UICollectionView {
         cell?.handleTapGesture(gesture)
     }
 
+    // NOTE: It's possible for small content size this wouldn't work - https://github.com/MessageKit/MessageKit/issues/725
+    public func scrollToLastItem(at pos: UICollectionView.ScrollPosition = .bottom, animated: Bool = true) {
+        guard numberOfSections > 0 else { return }
+        
+        let lastSection = numberOfSections - 1
+        let lastItem = numberOfItems(inSection: lastSection) - 1
```

Comment:
Consider renaming this to `lastItemIndex`. I think it better reflects what it is.

Reply:
Updated!